src/commands/txCommands/util/index.js

```diff
@@ -208,7 +210,13 @@ const uploadTrnsMaster$ = ([lang_tag, content]) =>
 		MASTER_RESOURCE,
 		lang_tag,
 		resourceContent(MASTER_RESOURCE, content)
-	);
+	).map(response => [lang_tag,response])
+	.do(([lang_tag,response]) => {
+			console.log(lang_tag);
+			console.log(response);
```

Comment:
what is expected here? would it be useful to do something like

```js
console.log('Upload response:', response);
```
?

Reply:
this is the output it produces - 

```
it-IT
{
    "strings_added": 0,
    "strings_updated": 600,
    "strings_delete": 0,
    "redirect": "/meetup/mup-web-master/master/"
}
es
{
    "strings_added": 0,
    "strings_updated": 600,
    "strings_delete": 0,
    "redirect": "/meetup/mup-web-master/master/"
}
```